### PR TITLE
Snort DB GID:SID link on the Alerts page. Implements #12221

### PR DESCRIPTION
**Pull request by vktg**
- [X] Redmine Issue: https://redmine.pfsense.org/issues/12221
- [X] Ready for review

### CODE REVIEW

File: security/pfSense-pkg-snort/files/usr/local/www/snort/snort_alerts.php [hunk 1]
```diff
@@ -1084,7 +1087,7 @@ function snort_match_filter_field($flent, $fields, $exact_match = FALSE) {
 				<td><?=$alert_src_p; ?></td>
 				<td style="word-wrap:break-word; white-space:normal"><?=$alert_ip_dst;?></td>
 				<td><?=$alert_dst_p; ?></td>
-				<td><?=$alert_sid_str; ?><br/><?=$sidsupplink; ?>&nbsp;&nbsp;<?=$sid_dsbl_link; ?>&nbsp;&nbsp;<?=$sid_action_link; ?></td>
+				<td><a target="_blank" href=<?=$link_sid_str; ?>><?=$alert_sid_str; ?></a><br/><?=$sidsupplink; ?>&nbsp;&nbsp;<?=$sid_dsbl_link; ?>&nbsp;&nbsp;<?=$sid_action_link; ?></td>
```
Comment (jim-p) [inline security/pfSense-pkg-snort/files/usr/local/www/snort/snort_alerts.php:1090]: `href` target should be surrounded by double quotes, so `<a target="_blank" href="<?=$link_sid_str?>">`

**Review by jim-p — CHANGES_REQUESTED** [review]

File: security/pfSense-pkg-snort/files/usr/local/www/snort/snort_alerts.php (hunk 1 above)
Comment (vktg) [reply security/pfSense-pkg-snort/files/usr/local/www/snort/snort_alerts.php]: Fixed

**Review by jim-p — APPROVED** [review]

**Comment by bmeeks8** [comment]
I approve this change. Thank you Viktor for the feature. I like it!

**Comment by rbgarga** [comment]
@vktg please bump port version or revision

**Comment by vktg** [comment]
> @vktg please bump port version or revision

Done

**Pull request merged**